scripts/mail-all-activity.py

```diff
@@ -1,181 +0,0 @@
-#!/usr/bin/env python
```

Comment:
Why is this one being cut? We used it just last week to confirm people's activity in the run-up to the election. Can't integrate into AMY - the HTTP request would time out.